### DISCUSSION TITLE
STM32 Nucleo-F103RB blinky issue

### DISCUSSION THREAD

**Discussion by Xman2008p** [zephyrproject-rtos/zephyr#77000] [General]
**Describe the bug**

<< Please describe the issue here >>
My borad is STM32 Nucleo-F103RB

It's my 1st time using ZpythrBuild the Blinky Sample and download.
And then LD1 keeping Red, LD2 is OFF.

Until I unplugged the USB port of F103 and connected it to the power bank.
LD1 is Blinking Red, LD2 is OFF.

My imaging is LD2 Blinking.
Because LD1 is indicator that the F103 board is downloading status. 

<< You may also want to update the automatically generated issue title above. >>

**Environment**

* Page: `develop/getting_started/index`
* Version: main
* SHA-1: 33d92b4acdab6e6891ae7a06dc86577d759bfc8e

**Comment by FRASTM** [comment]
The nucleo_f103rb has only one user led : this is a green led named LD2 on the schematics, connected to the GPIO pin PA5
The other led is LD1    dedicated to the stlink  communication. For more detail please look at the nucleo f103 [User Manual](file:///local/home/frq09495/Downloads/um1724-stm32-nucleo64-boards-mb1136-stmicroelectronics.pdf) 

Then if your board is not powered by the STlink (through CN1) but by another source VIN or E5V, you also need to change the JP5 position
(see § 6.3.2 External power supply inputs: VIN and E5V)
